### ISSUE TITLE
ENH: output format geoparquet

### ISSUE THREAD

**Issue by jsignell** [developmentseed/tipg#95]
This came up in the STAC meeting today. Currently it looks like the supported output formats (available with the `f` query param) are: 'geojson', 'html', 'json', 'csv', 'geojsonseq', 'ndjson'. I got that list by naively trying https://firenrt.delta-backend.com/collections/public.eis_fire_lf_perimeter_nrt/items?f=geoparquet

It would be neat to add 'geoparquet' as an option. 

Not sure if this is the right place to capture the request so feel free to close, just wanted to increase the visibility of that conversation.

**Comment by vincentsarago** [comment]
thanks for starting the discussion @jsignell 🙏 

This is definitely something we could/should support.

**Comment by jsignell** [comment]
yeah exactly! I think `/items` under just a different `f` query param. stac-geoparquet will definitely be helpful, but might need some alterations since these aren't STAC objects exactly.

**Comment by vincentsarago** [comment]
@kylebarron yes, I think it makes senses to enable GeoParquet output for `Items` first. (we might want collections later but it will be less useful). 

> It might be easiest to use the existing https://github.com/stac-utils/stac-geoparquet library for this

stac-geoparquet, depends on pandas and geopandas (thus shapely), this would be quite heavy dependencies `just` to add an output format. I was hopping for a more lightweight solution 🙏

**Comment by vincentsarago** [comment]
🤯 I don't think this feature is extremely needed right now so we can wait especially if this can help for your library to be ready :-) 

FYI: we already have pyproj dependency (via morecantile) 

Note: we could still add an heavy dependency and make the whole thing optional if this is really something user/customers want

**Comment by kylebarron** [comment]
Definitely agree with making it an optional dependency if we add it.

**Comment by bitner** [comment]
Yes, definitely would want to add this as optional dependency. We would probably want to implement this starting from a query like `SELECT column_a, column_b, ST_ASWKB(geometry_column) FROM mytable` rather than going through the geojson that we create and build up the geoparquet from those results which would eliminate the need for shapely or the like.

**Comment by kylebarron** [comment]
I'm not too familiar with the tipg internals but happy to help implement this
